.github/workflows/test_onnxruntime_gpu.yml

```diff
@@ -51,14 +51,12 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-      - name: Install dependencies
+      - name: Build image
         run: |
-          sudo apt -y update && sudo pip install --upgrade pip
-          pip install .[onnxruntime-gpu,tests]
-      - name: Test with unittest
-        working-directory: tests
+          docker build -f tests/onnxruntime/Dockerfile_onnxruntime_gpu -t onnxruntime-gpu .
+      - name: Test with unittest within docker container
         run: |
-          python3 -m unittest discover -s onnxruntime -p 'test_*.py'
+          docker run --gpus all --workdir=/workspace/optimum/tests onnxruntime-gpu:latest
```

Comment:
Do you need to do that? I mounted and run `run.sh` in the training image because I couldn't configure `torch_ort` while building the image.

Reply:
Never mind, it's better to do it your way.